docs/README.md

```diff
@@ -6,18 +6,18 @@ The Fluent Assertions [landing page](https://fluentassertions.com) is using Jeky
 
 ### Prerequisites
 
-* Ruby 2.4.x (note: 2.5 and higher may fail due to the `ffi` ruby lib only supporting < 2.5).
-  * An easy way to install is to use `choco install ruby --version 2.4.3.1`.
-  * Or use the **Ruby+Devkit installer** from [RubyInstaller for Windows](https://rubyinstaller.org/downloads/archives/).
-* The `bundler` gem (`gem install bundler`). If you receive SSL-related errors while running `gem install`, try running `refreshenv` first.
+* Ruby 3.1. 
+    * An easy way to install is to use `choco install ruby`.
+    * Or use the **Ruby+Devkit installer** from [RubyInstaller for Windows](https://rubyinstaller.org/downloads/archives/).
```

Comment:
Just for the record, I had to perform additional steps to get the same `Gemfile.lock`

* Install rubyinstaller-devkit-3.1.0-1-x64.exe (Didn't get the choco installed ruby to work with MSYS2 within 5 minutes)
* Ran `gem update --system` to get bundler 2.3.7